### DISCUSSION TITLE
fldigi to not1mm mode contest  rtty

### DISCUSSION THREAD

**Discussion by ik3ocd** [mbridak/not1mm#238] [General]
hi, sorry for the inconvenience, I'm trying to do not1mm and fldigi activities by following the instructions, I can't get fldigi-->not1mm to communicate, xlmlprc it's active, what's missing to configure on not1mm, please can you help me with the correct configuration of the 2 programs so that they communicate, thanks for your courtesy, 73 flavio ik3ocd

**Comment by mbridak** [comment]
not1mm sends xmlrpc commands to fldigi's standard address/port 127.0.0.1:7362.
If you need to change the xmlrpc port and address you can do it thru command line arguments when launching fldigi.

```text
--xmlrpc-server-address HOSTNAME
Set the XML-RPC server address. Defaults to “127.0.0.1”.

--xmlrpc-server-port PORT
Set the XML-RPC server port. Defaults to “7362”.
```

**Comment by mbridak** [comment]
Just changed the way QSO's are detected when logged in fldigi. 

switch from watching the fldigi log file, to monitoring the fldigi ADIF UDP traffic.

[See](https://github.com/mbridak/not1mm?tab=readme-ov-file#logging-wsjt-x-ft8ft4etc-and-fldigi-rtty-contacts)

**Comment by ik3ocd** [comment]
hi michael, thanks for your kind help for this great project of yours. 73
de ik3ocd Flavio .

Il Dom 15 Dic 2024, 01:03 Michael Bridak ***@***.***> ha
scritto:
